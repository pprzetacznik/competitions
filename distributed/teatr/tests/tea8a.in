100000000 3 32 19 1000000 7419359541164124855
