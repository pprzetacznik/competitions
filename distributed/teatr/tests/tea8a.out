1618556645436950
